Mathlib/Data/Set/Insert.lean

```diff
@@ -347,6 +347,10 @@ theorem disjoint_insert_right : Disjoint s (insert a t) ↔ a ∉ s ∧ Disjoint
   obtain ⟨y, hy⟩ := exists_ne x
   exact ⟨y, by simp [hy]⟩
 
+@[simp]
+theorem singleton_ne_univ [Nontrivial α] (a : α) : {a} ≠ univ :=
```

Comment:
Can you upgrade this to ssubset? It's easy to get ne using [HasSSubset.SSubset.ne](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Order/RelClasses.html#HasSSubset.SSubset.ne).

Reply:
I'll do it tonight.